coti.py

```diff
@@ -295,20 +277,26 @@ def bonanza():
     return Decimal(compra), Decimal(venta)
 
 
-# def familiar():  # Comentado porque el servidor bloquea las peticiones
-#     try:
-#         soup = BeautifulSoup(
-#             requests.get('https://www.familiar.com.py/', timeout=10).text, "html.parser")
-#         compra = soup.select(
-#             'hgroup:nth-of-type(1) > div:nth-of-type(2) > p:nth-of-type(2)')[0].get_text().replace('.', '')
-#         venta = soup.select(
-#             'hgroup:nth-of-type(1) > div:nth-of-type(3) > p:nth-of-type(2)')[0].get_text().replace('.', '')
-#     except requests.ConnectionError:
-#         compra, venta = 0, 0
-#     except:
-#         compra, venta = 0, 0
+# Uses html5lib parser instead of Beautiful soup because
+# BeautifulSoup parser dies because banco Familiar HTML is malformed
```

Comment:
I was about to comment about that we don't need another html parser because we use bs4 but now I read this 🤦🏻‍♂️
lol